config/crds/edge.kubestellar.io_customizers.yaml

```diff
@@ -3,8 +3,9 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.10.0
-  creationTimestamp: null
```

Comment:
Why was this line here (and in the other YAMLs)? Does removing it break something?

Reply:
As far as what I saw in local test and the CI, I didn't see it break anything. I think it is introduced by the more recent version of kubebuilder.

Reply:
Yes